build_tools/benchmarks/suites/cmake_rule_generator.py

```diff
@@ -18,22 +18,22 @@
 from .definitions import common_definitions, iree_definitions
 from . import iree_benchmarks
 
-TEMPLATE_DIR = pathlib.Path(__file__).parent
-
 
-def read_template_from_file(template_name: str) -> string.Template:
-  with open(TEMPLATE_DIR / template_name, "r") as f:
+def read_template_from_file(template_dir: pathlib.Path,
```

Comment:
Do we want the templates to be in different directories? Not sure if that's more or less clear. Given that you're now passing the directory though, I think it makes more sense to just accept a path. Directory + basename is a weird API

Reply:
Ah.. makes sense. Done.